clarified instructions for certificates stored in hardware (#3276)
NuGet/docs.microsoft.com-nuget: docs/create-packages/Sign-a-Package.md

## Export the certificate file

> * If your certificate is stored in a hardware token, you don't need to export the certificate. Instead, specify the SHA-1 certificate fingerprint (thumbprint) by using the option `--certificate-fingerprint <SHA-1fingerprint>` (replacing `--certificate-path <PathToTheCertificate>`)

Reviewer: ```suggestion
* If your certificate is stored in a hardware token, you should not export the certificate. Instead, specify the SHA-1 certificate fingerprint (thumbprint) by using the option `--certificate-fingerprint <SHA-1fingerprint>` (replacing `--certificate-path <PathToTheCertificate>`)
```
